app/src/main/res/values/strings.xml

```diff
@@ -981,7 +981,7 @@
     <!-- text for the firefox account onboarding card header when we detect you're already signed in to
         another Firefox browser. (The word `Firefox` should not be translated)
         The first parameter is the email of the detected user's account -->
-    <string name="onboarding_firefox_account_auto_signin_header_2">You are signed in as %s on another Firefox browser on this phone. Would you like to sign in with this account?</string>
```

Comment:
Thanks for adding this as a new string id so it will get translated!

One thing that we're doing is quarantining string removals, since our localization will be happening during the Beta period, so new strings will need to stick around bc the new translations land in Nightly and need to be uplifted. See https://github.com/mozilla-mobile/fenix/issues/14176